README.md

```diff
@@ -17,18 +17,41 @@
 </div>
 <br/>
 
-This crate provides the ability to annotate structs with a `#[derive(Inspectable)]`,
-which opens a debug interface using [egui](https://github.com/emilk/egui) where you can visually edit the values of your struct live.
-
-Your struct will then be available to you as a bevy resource.
+This crate provides a debug interface using [egui](https://github.com/emilk/egui) where you can visually edit the values of your components live.
 
 <img src="./docs/inspector.jpg" alt="demonstration with a running bevy app" width="500"/>
 
+## Usage
 
-More examples (with pictures) can be found in the [`examples folder`](examples).
+In order for custom components to show up in the inspector, you have to:
+
+1. Add an `InspectableRegistry` to the world as a resource
+2. Register the component to the `InspectableRegistry`
+3. `#[derive(Inspectable)]` for the component
+4. Add an inspector plugin
```

Comment:
If you add the `WorldInspectorPlugin` first, you don't have to add an `InspectableRegistry`, you can just add to the existing one. I think the order would be either
- only display a single struct? Add an `InspectorPlugin<T>`
- want to use the WorldInspector?
1. add the `WorldInspectorPlugin`
2. `#[derive(Inspectable)]`
3. call `.register::<T>()` on the `InspectbleRegistry`.

Reply:
Is that better?

Reply:
Yup thats good